### ISSUE TITLE
StarTracker not working

### ISSUE THREAD

**Issue by SeppeStaelens** [SeppeStaelens/GRChombo#5]
Including the StarTracker in the run gives an error
```
terminate called after throwing an instance of 'std::invalid_argument'
     what(): stod
```
which seems to refer to some error converting a string to a double.

Everything works at t=0, but then breaks once the startracker is called again.

**Comment by SeppeStaelens** [comment]
Found approximately what went wrong -- by trying to change the .dat file, such that it would be outputed in the data subfolder, I called two  separate instances of write_header - like functions, causing x_1, y_1, z_1, part of the actual header, to be printed on line 2. This then caused the stod error when 'x_1' was interpreted as double.

Trying to fix that now

**Comment by SeppeStaelens** [comment]
Fixed in PR [#6](https://github.com/SeppeStaelens/GRChombo/pull/6)
